ts/components/session/registration/SignInTab.tsx

```diff
@@ -80,14 +80,17 @@ const SignInButtons = (props: {
     <div>
       <RestoreUsingRecoveryPhraseButton onRecoveryButtonClicked={props.onRecoveryButtonClicked} />
       <SpacerLG />
-      <div className="or">{window.i18n('or')}</div>
-      <SpacerLG />
       <LinkDeviceButton onLinkDeviceButtonClicked={props.onLinkDeviceButtonClicked} />
     </div>
   );
 };
 
-export const SignInTab = () => {
+interface Props {
+  setRegistrationPhase: (phase: any) => any;
```

Comment:
you can probably set the type here:
`setRegistrationPhase: (phase: RegistrationPhase) => void; `